src/test/java/org/alliancegenome/curation_api/ExpressionBulkUploadFmsITCase.java

```diff
@@ -0,0 +1,120 @@
+			.body("results[0].dataProvider.sourceOrganization.abbreviation", is("ZFIN"))
+			.body("results[0].expressionAnnotationSubject.modEntityId", is(gene))
+			.body("results[0].expressionAssayUsed.curie", is(mmoTerm))
+			.body("results[0].whereExpressedStatement", is("trunk"))
+			.body("results[0].whenExpressedStageName", is("stage1"))
+			.body("results[0].singleReference.crossReferences[0].referencedCurie", is(publicationId))
+			.body("results[0].relation.name", is(VocabularyConstants.GENE_EXPRESSION_RELATION_TERM));
+	}
+
+	@Test
+	@Order(2)
+	public void expressionBulkUploadMissingRequiredFields() throws Exception {
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_01_no_geneId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_02_no_dateAssigned.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_03_no_evidence.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_04_no_assay.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_05_no_whenExpressed.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "MR_06_no_whereExpressed.json");
+	}
+
+	@Test
+	@Order(3)
+	public void expressionBulkUploadEmptyRequiredFields() throws Exception {
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_01_empty_geneId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_02_empty_dateAssigned.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_03_empty_crossReferenceId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_04_empty_assay.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_05_empty_whenExpressed.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "ER_06_empty_whereExpressed.json");
+	}
+
+	@Test
+	@Order(4)
+	public void expressionBulkUploadInvalidFields() throws Exception {
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_01_invalid_geneId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_02_invalid_dateAssigned.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_03_invalidCrossReferenceId.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_04_invalid_assay.json");
+		checkFailedBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "IV_05_invalid_publicationId.json");
+	}
+
+	private void loadRequiredEntities() throws Exception {
+		DataProvider dataProvider = createDataProvider("ZFIN", false);
+		Vocabulary vocabulary1 = getVocabulary(VocabularyConstants.NAME_TYPE_VOCABULARY);
+		VocabularyTerm symbolTerm = getVocabularyTerm(vocabulary1,"nomenclature_symbol");
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.